tests/unit/models/modelRequestBody.test.js

```diff
@@ -0,0 +1,60 @@
+const modelRequestBody = require('../../../api/model/modelRequestBody');
```

Comment:
`modelSimplePatron` tests all the separate functions found in `modelRequestBody` which is fine, but I also think individual tests for the functions should also have tests, such as `updateDateOfBirthToBirthdate `.

Reply:
Sounds good, @EdwinGuzman.  I'd be interested in a quick chat about this.  Coming from a Rails background, I follow Sandi Metz' philosophy of testing only the public methods; in a Node context, I interpret that as only the methods that are exported.  In that philosophy, the key is to test each branch of the code (including the private/non-exported methods) which is what I think I was able to do with the different variations of inputing birthday data.

This methodology gives a developer the flexibility to refactor or change the name of `updateDateOfBirthToBirthdate` while not requiring a rewrite of the tests as long as they didn't break any functionality.

But if our standard is to test the private/non-exported methods, then I certainly will!  

Since we're at the end of the sprint, I'd suggest we merge this (if everything else looks good) and we add a new ticket for tests on private methods next sprint.